crates/sozo/ops/src/migration/utils.rs

```diff
@@ -30,6 +31,20 @@ where
     let mut local_manifest = BaseManifest::load_from_path(&profile_dir.join(BASE_DIR))
         .map_err(|e| anyhow!("Fail to load local manifest file: {e}."))?;
 
+    if let Some(skip_manifests) = skip_migration {
```

Comment:
Actually after reading this in lot's of places, would it be convenient to have a type to manage that? Which could have a function implemented once, and only called when required?

Reply:
good idea, added it directly in  `BaseManifest`